app/longest_strings.js

```diff
@@ -0,0 +1,15 @@
+exports.longestString = function(array) {
+  if (array[0] === undefined) {
+    return null;
+  }
+  var firstLength = array[0].length;
```

Comment:
Like that you started with the first word! I would start the loop from `var i=1` since you already have the first word